### PR TITLE
Add aggregations to Search-Hybrid search section

### PR DESCRIPTION
**Pull request by martin-gaievski**
### Description
Extend section for Hybrid search by adding information and examples for aggregations. 

### Issues Resolved
Closes https://github.com/opensearch-project/documentation-website/issues/6644

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by kolchfa-aws — APPROVED** [review]
Thanks so much for providing this documentation, @martin-gaievski! The doc review is complete. I'll take the PR from here through editorial review.

File: _search-plugins/hybrid-search.md [hunk 1]
```diff
@@ -216,3 +217,196 @@ The response contains the matching document:
...
 }
 ```
+
+## Step 6 (Optional): Enhance hybrid search results
+
+You can enhance search results by combining a hybrid query clause with any aggregation that OpenSearch supports. Aggregations allow you to use OpenSearch as an analytics engine. For more information about aggregations, see [Aggregations]({{site.url}}{{site.baseurl}}/aggregations/).
+
+Most aggregations are performed on the subset of documents that is returned by a hybrid query. The only aggregation that operates on all documents is the [`global`]({{site.url}}{{site.baseurl}}/aggregations/bucket/global/) aggregation.
+
+To illustrate how aggregations can be used with hybrid query, first let's create an index. Aggregations are typically used on fields of special types, like `keyword` or `integer`. The following example creates an index with several such fields:
```
Comment (natebower) [inline _search-plugins/hybrid-search.md:227]:
```suggestion
To illustrate how aggregations can be used with a hybrid query, first let's create an index. Aggregations are typically used on fields of special types, like `keyword` or `integer`. The following example creates an index with several such fields:
```

Comment (natebower) [inline _search-plugins/hybrid-search.md:227]: Can we remove or replace "let's"?

File: _search-plugins/hybrid-search.md [hunk 2]
```diff
@@ -216,3 +217,196 @@ The response contains the matching document:
...
+      "category": {
+        "type": "keyword"
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+The following request ingests six documents to your new index:
```
Comment (natebower) [inline _search-plugins/hybrid-search.md:252]:
```suggestion
The following request ingests six documents into your new index:
```

**Review by natebower — COMMENTED** [review]
@martin-gaievski @kolchfa-aws Just a few minor changes. Thanks!

**Pull request merged**